doc/source/reference/io.rst

```diff
@@ -127,6 +127,13 @@ SQL
    read_sql_query
    read_sql
 
+.. currentmodule:: pandas.io.sql
```

Comment:
I think just do `sql.get_schema` in the block above. This shouldn't need its own section.